python/MooseDocs/common/read.py

```diff
@@ -49,4 +49,6 @@ def get_language(filename):
         return 'cpp'
     elif ext == '.py':
         return 'python'
+    elif ext == '.i':
+        return 'moose'
```

Comment:
Does this work for test specs too? Should be able to:

```suggestion
    elif ext in ['.i', 'tests']:
        return 'moose'
```

Reply:
No it doesn't. That would require another "language" to be defined in prism.js. The vscode extension does have a separate language definition for test specs, I believe.

Reply:
Future thing - if you want.